src/routes.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Route } from 'react-router-dom';
+
+import App from './components/App';
+import Signin from './containers/SignIn';
+
+/**
+ * @name Routes
+ * @summary Renders the application routes
+ * @return {jsx} React node for the application routes
+ */
+const Routes = () => (
+  <div>
```

Comment:
This could be browser router.

Reply:
A variable already exist with the name browser router, I think router would be fine as well. What do you think?